src/java.desktop/share/classes/java/awt/image/BufferedImage.java

```diff
@@ -1576,13 +1587,23 @@ public boolean isTileWritable (int tileX, int tileY) {
      * Returns an array of {@link Point} objects indicating which tiles
      * are checked out for writing.  Returns {@code null} if none are
      * checked out.
+     * <p>
+     * Since a {@code BufferedImage} consists of a single tile,
+     * and that tile is always checked out for writing, the
+     * default implementation returns an array of one point.
```

Comment:
I'd be happier with "the default implementation" -> "this method".
I did read your notes on subclasses etc but really these methods should have been final so I'd like to make
the documentation reflect that rather than implying that there's an open door to do something else.

Reply:
Replaced all "The default implementation" texts by "This method". But if the intend was to disallow alternative behaviour, maybe instead of making those method final, `BufferedImage` should have simply not implemented `WritableRenderedImage` and implement `RenderedImage` instead?